### PR TITLE
Disable compile grpo loss

### PR DESCRIPTION
**Pull request by samsja**

### CODE REVIEW

**Review by Jackmin801 — APPROVED** [review]
lfgtm. Why the commenting out jaxtyping though?

**Comment by samsja** [comment]
> lfgtm. Why the commenting out jaxtyping though?

ah good catch, I will out comment the comment out. I thought for a minute that the jaxtyping was casting my tensor but probably wrong assumtpion

File: src/zeroband/training/loss.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import torch
 from torch import Tensor
-from jaxtyping import Float, Int, jaxtyped
-from beartype import beartype as typechecker
+from jaxtyping import Float, Int
 import torch.nn.functional as F
+from jaxtyping import jaxtyped
+from beartype import beartype as typechecker
```
Comment (Jackmin801) [inline src/zeroband/training/loss.py]: These imports resolve to the same thing. samsja here pumping his contribution numbers tsk tsk

Comment (samsja) [reply src/zeroband/training/loss.py]: fuck haha
